test/README.md

```diff
@@ -0,0 +1,56 @@
+## Reproducibility Pytests
+
+These pytests are used for reproducibility CI checks in the [ACCESS-NRI/reproducibility](https://github.com/ACCESS-NRI/reproducibility) repository. Code from these tests is adapted from COSIMAS's ACCESS-OM2's [
+bit reproducibility tests](https://github.com/COSIMA/access-om2/blob/master/test/test_bit_reproducibility.py).
+
+### How to run tests manually
+
+1. First clone the pytest code which is on the `main` branch of `access-om2-configs` repository into a separate directory.
+```sh
+git clone https://github.com/ACCESS-NRI/access-om2-configs/ test-code
+```
+
+2. Checkout an experiment
+```sh
+git clone https://github.com/ACCESS-NRI/accessom2-configs/ <experiment>
+cd <experiment>
+git checkout <branch/tag>
+```
+
```

Comment:
Note the suggestion is a bit broken because of multiple nested triple backquotes not working well.

Reply:
I didn't use `payu clone` so to disable generating metadata, new UUIDs and creating an archive in the /scratch/tm70/$USER/$MODEL default lab directory. I also wanted to keep the experiment name simple (without the UUID) as otherwise would need to parse the metadata to find the archive.

This also required setting the following for each experiment run in the pytests in `config.yaml`:
```
# Disable metadata and set override experiment name for work/archive
# directories
doc['metadata'] = {"enable": False}
doc['experiment'] = self.exp_name # Name of experiment

# Set laboratory path - needed for auto-collate job
doc['laboratory'] = str(self.lab_path)
```

Reply:
Fair enough. 

It would be nice to make `payu` more amenable to being driven programmatically, for testing but also for generating experiments programmatically, e.g. parameter sweeps.

I have made an issue to collect these uses cases together.

https://github.com/payu-org/payu/issues/417